daniellamarr/merry-sims-test
# #162767854: Signup and Email verification

#### What does this PR do?
Signup and Email verification

#### Description of Task to be completed?
- Users can sign up locally to the application
- User gets Link to verify email provided on signup
- User clicks on the verify link and verify the email
- Updates database for verified email

#### How should this be manually tested?
- Clone the repo and install the npm packages
-  Create database and run migration
- Start server on port `http://localhost:9090`
- Signup providing the required fields `name, email, password, bio, image` on `http://localhost:9090/api/v1/auth/users/signUp`
- Check email for link and click to verify

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/162767854

#### Screenshots (if appropriate)
